kernel/update_kernel.sh

```diff
@@ -17,62 +17,58 @@ VERSION=${1:-$clear_vm_kernel_version}
 OBS_PUSH=${OBS_PUSH:-false}
 OBS_CC_KERNEL_REPO=${OBS_CC_KERNEL_REPO:-home:clearlinux:preview:clear-containers-staging/linux-container}
 
-git checkout obs_working_dir/debian/changelog
-last_release=`cat obs_working_dir/debian/changelog | head -1 | awk '{print $2}' | cut -d'-' -f2 | tr -d ')'`
+git checkout debian.changelog
+last_release=$(cat debian.changelog | head -1 | awk '{print $2}' | cut -d'-' -f2 | tr -d ')')
 next_release=$(( $last_release + 1 ))
+kernel_sha256=$(curl -s  https://cdn.kernel.org/pub/linux/kernel/v4.x/sha256sums.asc | grep linux-${VERSION}.tar.xz | awk '{print $1}')
```

Comment:
What about 
`awk '/linux-'${VERSION}'.tar.xz/ {print $1}'`
instead of 
`grep linux-${VERSION}.tar.xz | awk '{print $1}'`